### ISSUE TITLE
No balance at all after update to 2.5.3

### ISSUE THREAD

**Issue by sargatxet** [iotaledger/wallet#412]
Tryed to reclaim and creating new receive addesss, but balance remains = 0. Any suggestion?

**Comment by sargatxet** [comment]
Just used old seed, selected RECEIVE and ATTACH TO TANGLE. Received all my balance. Thanks.

**Comment by POLOC** [comment]
EFECTIVAMENTE! NO ENCONTRABA MIS IOTAs y al realizar la selección de recibir y vincular al tangle (con la antigua semilla) me han aparecido

**Comment by jogearg** [comment]
misma situacion que POLOC y sargatxet
same situation and it works

**Comment by maliza001** [comment]
My balance is still showing $0   HELP!   I did the old seed and the new seed  but still nothing in my wallet

**Comment by POLOC** [comment]
maliza001, haz lo que puso sargatxet, bajate la billetera 2.5.3 y mete la semilla antigua, puede que inicialmente te 0 de saldo pero pica en RECIBIR y da VINCULAR AL TANGLE, (el recuadro morado) y sigue haciéndolo hasta que te salga tu saldo positivo con TUS IOTAS

**Comment by POLOC** [comment]
como te digo, baja la billetera 2.5.3, mete semilla antigua, pica en RECIBIR y pica en el rectángulo morado hasta que recuperas TU IOTAS, a nosotros tres nos ha funcionado  suerte!!!

**Comment by maliza001** [comment]
ok did as you said (several times)  I had a lot more IOTA but I only got  1.015 GI   Where did the rest go?

**Comment by POLOC** [comment]
Deberías recuperar todas tus IOTA. Yo recuperé todas mis IOTA al realizar esa operación,  ya no sé que decirte que hagas...

**Comment by turk1974** [comment]
same issue here, still ZERO after I have completed everything said in IOTA site. Won't IOTA help us? this is like our money disappearing from the bank and bank is not doing anything to recover.  how do we get help from IOTA directly?

**Comment by POLOC** [comment]
turk1974, haz lo que hemos hecho los demás. Baja la wallet 2.5.3 y mete la semilla antigua (la que tenías cuando creaste tu wallet inicial) y luego presiona en recibir hasta que te salga tu saldo y luego presiona en el recuadro morado (vincular al tangle), debería de aparecerte tu saldo.
